src/retest/main.c

```diff
@@ -272,6 +270,17 @@ xopen(const char *s)
 	return f;
 }
 
+static void
+xclock_gettime(struct timespec *tp)
+{
+	assert(tp != NULL);
+
+	if (clock_gettime(CLOCK_MONOTONIC, tp) != 0) {
+		perror("clock_gettime");
+		exit(EXIT_FAILURE);
+	}
```

Comment:
Do we need to `abort()` here?  If there's no timing info, that's okay.  retest isn't really about measuring performance.

I'm ambivalent.  This is certainly easier logic to follow.